### ISSUE TITLE
app stuck in splash screen on ios 16 and 17

### ISSUE THREAD

**Issue by gustavohpaula** [oracle/cordova-plugin-wkwebview-file-xhr#93]
I have an application that uses cordova-ios@5.1.1. When running the app on iOS 16 or 17, the app crashes on the splash screen without any log on xcode and the safari don't recognize it to see the js log.
During the test I find that the problem is becaus of the dependency of the WKWebViewEngine. 
I've tried to upgrade the cordova-ios to version 6 but the WKWebViewEngine is integrated on the core, so the problem persist

**Comment by jurahalas** [comment]
@gustavohpaula Have you manage to fix the issue?

**Comment by gustavohpaula** [comment]
yes, it was the cordova-plugin-screen-orientation. This plugin was in conflict with the others, which is why the app stuck
